src/Razor/src/Microsoft.AspNetCore.Razor.Language/ComponentResources.resx

```diff
@@ -117,6 +117,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AttributeDirective_AttributeToken_Description" xml:space="preserve">
+    <value>The C# attribute that will be applied to the current document.</value>
+  </data>
+  <data name="AttributeDirective_AttributeToken_Name" xml:space="preserve">
+    <value>Attribute</value>
+  </data>
+  <data name="AttributeDirective_Description" xml:space="preserve">
+    <value>Specifies the C# attribute that will be applied to the current document.</value>
```

Comment:
This should say that it applies to the generated class. I think it's important to specifcially say that it goes on the class, even those that's obvious to us.

Reply:
Same [question](https://github.com/aspnet/AspNetCore-Tooling/pull/607#discussion_r286153299) as above.